tests/posix/mt/mt.cpp

```diff
@@ -106,47 +96,29 @@ create_close_unlink_worker(void *arg)
 		}
 		pmemfile_unlink(global_pfp, path);
 	}
-
-	return NULL;
 }
 
 TEST_F(mt, open_close_create_unlink)
 {
-	int i = 0;
-	int ret;
-
-	for (int j = 0; j < ncpus / 2; ++j) {
-		ret = pthread_create(&threads[i++], NULL, open_close_worker,
-				     (void *)"/aaa");
-		ASSERT_EQ(ret, 0) << strerror(errno);
-		ret = pthread_create(&threads[i++], NULL,
-				     create_close_unlink_worker,
-				     (void *)"/aaa");
-		ASSERT_EQ(ret, 0) << strerror(errno);
+	for (unsigned j = 0; j < ncpus / 2; ++j) {
```

Comment:
Are we sure this should be limited by number of cpus?
There can be race issues with multiple threads on a single core as well, as I recall.